diff -uNr fvwm-crystal-1.0.2-orig/bin/fvwm-crystal.install fvwm-crystal-1.0.2/bin/fvwm-crystal.install
--- fvwm-crystal-1.0.2-orig/bin/fvwm-crystal.install	2004-03-18 21:36:16.000000000 +0100
+++ fvwm-crystal-1.0.2/bin/fvwm-crystal.install	2004-05-24 15:37:32.384111864 +0200
@@ -9,7 +9,7 @@
 # use 'fvwm-crystal.install' script instead.
 
 ## Variables
-DATADIR=/usr/local/share/fvwm-crystal
+DATADIR=/usr/share/fvwm-crystal
 CONFIG_PATH=${DATADIR}/config/
 CONFIG_INSTALL_PATH="$HOME/.fvwm"
 SESSION_FILE_PATH="$HOME/.Xsession"
